companion.py

```diff
@@ -952,6 +955,9 @@ def capi_station_queries(  # noqa: CCR001
                 if query.endpoint == self._CAPI_PATH_STATION:
                     capi_data = capi_station_queries(query.capi_host)
 
+                elif query.endpoint == self.FRONTIER_CAPI_PATH_FLEETCARRIER:
+                    capi_data = capi_single_query(query.capi_host, self.FRONTIER_CAPI_PATH_FLEETCARRIER)
```

Comment:
This needs to specify a custom `timeout=...`, because the default 10s isn't going to work for the slow queries.

The query is threaded, so won't hold up the EDMC UI, but can hold up other CAPI queries, so I'm inclined to try this with 60s for now.

Reply:
Long-term we might need to adjust `capi_query_worker()` to actually perform *these* (or future other 'not station' query set) queries in their own sub-thread.  But that can wait until we see how *this* shakes out in practice.